src/fidesops/models/privacy_request.py

```diff
@@ -91,6 +94,14 @@ class PrivacyRequest(Base):
         index=True,
         nullable=False,
     )
+    # When the request was approved/denied
+    reviewed_at = Column(DateTime(timezone=True), nullable=True)
+    # Who approved/denied the request
+    reviewed_by = Column(
+        String,
+        ForeignKey(FidesopsUser.id_field_path, ondelete="SET NULL"),
+        nullable=True,
+    )
```

Comment:
Currently, `PrivacyRequest.reviewed_by` is a FK to fidesopsuser.  Users can be deleted, which is an issue b/c of the FK.  I am currently just setting the `reviewed_by` field to None if the user is deleted, and leaving the privacy request in the db.